flink-cyber/flink-enrichment/flink-enrichment-combined/src/main/java/com/cloudera/cyber/enrichment/EnrichmentJob.java

```diff
@@ -59,6 +63,7 @@ public abstract class EnrichmentJob {
 
     private static final String PARAMS_ENABLE_GEO = "geo.enabled";
     private static final String PARAMS_ENABLE_ASN = "asn.enabled";
+    private static final String PARAMS_ENABLE_CIDR = "cidr.enabled";
     private static final String PARAMS_ENABLE_HBASE = "hbase.enabled";
```

Comment:
Also add a sample json config file similar to flink-cyber/cyber-jobs/src/main/resources/conf/templates/triage/enrichments-lookups.json to help the user get started with the configs.